nginx-fem-redirects.conf

```
@@ -31,135 +31,135 @@ location ~* ^/about/(?:team|publications)/?$ {
 }
 
 # FEM projects
-location ~* ^/projects/zookeeper/galaxy-zoo-weird-and-wonderful/?(?:(classify|about)(?:/.+?)?)?/?$ {
+location ~* ^/projects/[\w-]*?/*zookeeper/galaxy-zoo-weird-and-wonderful/?(?:(classify|about)(?:/.+?)?)?/?$ {
```

I think we can make the regex tighter to only ignore numbers and underscores and use an optional non-capture group (like the existing optional paths)

```suggestion
location ~* ^/projects/(?:[[:alpha:]-]*?/)?zookeeper/galaxy-zoo-weird-and-wonderful/?(?:(classify|about)(?:/.+?)?)?/?$ {
```

I've tested this locally and it seems to work matching lang codes like `/en/` or `zh-HK` but not `en_123`

---

Hmmm. IETF subcodes can be numbers eg. Latin-American Spanish is `es-419`, but I don't know if we'd ever use that in a URL.

---

if they are then skip the alpha only part `\w` will do.
```suggestion
location ~* ^/projects/(?:[\w-]*?/)?zookeeper/galaxy-zoo-weird-and-wonderful/?(?:(classify|about)(?:/.+?)?)?/?$ {
```

---

I've updated my test expression. Tests all pass.
https://regexr.com/6m2gu

I'll update the PR.